miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt

```diff
@@ -32,11 +32,18 @@ interface MiniAppFileChooser {
      * @param filePathCallback a callback to provide the array of file-paths to select.
      * @param fileChooserParams the parameters can be used to customize the options of file chooser.
      * @param context the Activity context can be used to start the intent to choose file.
+     * @param isOpenMultipleMode derived from WebChromeClient function,
+     * @param isCaptureEnabled derived from WebChromeClient function
+     * @param fileChooserParamsAcceptTypes derived from WebChromeClient function
      **/
+    @Suppress("LongParameterList")
     fun onShowFileChooser(
         filePathCallback: ValueCallback<Array<Uri>>?,
         fileChooserParams: WebChromeClient.FileChooserParams?,
-        context: Context
+        context: Context,
+        isOpenMultipleMode: Boolean = fileChooserParams?.mode == WebChromeClient.FileChooserParams.MODE_OPEN_MULTIPLE,
```

Comment:
Good work, just one suggestion,i think we don't need these parameters, as `onShowFileChooser` is a public interface, we should not add parameters that has no external use, please try refactor the code the way that we dont have to add extra parameters just for test case, thanks